src/Contracts/TasksQuery.php

```diff
@@ -63,6 +75,7 @@ public function toArray(): array
                     'from' => $this->from,
                     'limit' => $this->limit,
                     'canceledBy' => $this->formatArray($this->canceledBy),
+                    'reverse' => null !== $this->reverse ? json_encode($this->reverse) : null,
```

Comment:
https://github.com/meilisearch/meilisearch-php/blob/ef1769975bcc3b97057dc6ee3ed3f733a8ec7f0e/src/Contracts/DocumentsQuery.php#L119

avoid json_encode to be consistent :)

Reply:
Thank you for the suggestion!